client/src/pages/Messages/Messages.tsx

```diff
@@ -28,6 +32,11 @@ export default function Messages(): JSX.Element {
       setSearch('');
     }
   };
+  useEffect(() => {
+    if (socket) {
+      socket.emit('addUser', loggedInUser?.id);
```

Comment:
Maybe also check if a logged in user is available here?

Reply:
Successfully added suggested if case. 

commit 5f5ee0e5013612f4bc4fc708bbc8309c80820c4e